automation/run-tests.sh

```diff
@@ -18,27 +18,29 @@ TEST_TYPE_INTEG="integ"
 FEDORA_IMAGE_DEV="nmstate/fedora-nmstate-dev"
 CENTOS_IMAGE_DEV="nmstate/centos8-nmstate-dev"
 
+: ${container_cmd:=docker}
```

Comment:
Please use capital for environment variable

Reply:
will change this